ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__arm64-nightly-4.17-upgrade-from-stable-4.12.yaml

```diff
@@ -61,27 +66,30 @@ releases:
       architecture: arm64
       product: ocp
       stream: nightly
-      version: "4.16"
+      version: "4.17"
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-ipi-ovn-ipsec-f14
-  cron: 23 19 10,24 * *
+- as: baremetal-upi-ovn-ipv4-f28
```

Comment:
In the spreadsheet, it's baremetalds-ipi-ovn-ipv4 (arm). Please keep them consistent.